actionpack/lib/action_dispatch/routing/inspector.rb

```diff
@@ -82,11 +81,19 @@ def format(formatter, filter = nil)
 
       private
 
-      def filter_routes(filter)
-        if filter
+      def filter_routes(options)
```

Comment:
It's interesting idea but I don't agree with you. With my solution you have simple way for adding new search pattern (for example, in future, it will be great idea to add search on actions).